ambry-network/src/main/java/com.github.ambry.network/Selector.java

```diff
@@ -90,10 +92,12 @@ public Selector(NetworkMetrics metrics, Time time, SSLFactory sslFactory)
     this.completedReceives = new ArrayList<NetworkReceive>();
     this.connected = new ArrayList<String>();
     this.disconnected = new ArrayList<String>();
+    this.pendingSslTrans = new HashMap<>();
     this.metrics = metrics;
     this.IdGenerator = new AtomicLong(0);
     this.activeConnections = new AtomicLong(0);
-    this.metrics.initializeSelectorMetricsIfRequired(activeConnections);
+    this.connectionsPendingHandshakeCount = new AtomicLong(0);
+    this.metrics.initializeSelectorMetricsIfRequired(activeConnections, connectionsPendingHandshakeCount);
```

Comment:
Prefixing with `this` is unnecessary for most member variables in this constructor.